extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -861,16 +865,20 @@ export class AmpA4A extends AMP.BaseElement {
         // Non-AMP creative case, will verify ad url existence.
         return this.renderNonAmpCreative_();
       }
+      promiseId = this.promiseId_;
```

Comment:
Why is this necessary?  By definition it should still be the same value otherwise we should have cancelled.  Make promiseId a const